store/models/log_events.go

```diff
@@ -22,44 +22,61 @@ const (
 	RequestLogTopicAmount
 )
 
+const (
+	evmWordSize      = common.HashLength
+	idSize           = evmWordSize
+	versionSize      = evmWordSize
+	callbackAddrSize = evmWordSize
+	callbackFuncSize = evmWordSize
+	expirationSize   = evmWordSize
+	dataLocationSize = evmWordSize
+	dataLengthSize   = evmWordSize
+)
+
 var (
-	// RunLogTopic is the signature for the RunRequest(...) event
-	// which Chainlink RunLog initiators watch for.
-	// See https://github.com/smartcontractkit/chainlink/blob/master/solidity/contracts/Oracle.sol
-	RunLogTopic = utils.MustHash("RunRequest(bytes32,address,uint256,uint256,uint256,bytes)")
+	// RunLogTopic0 was the original topic to filter for Oracle.sol RunRequest events.
+	RunLogTopic0 = utils.MustHash("RunRequest(bytes32,address,uint256,uint256,uint256,bytes)")
+	// RunLogTopic20190123 was the new RunRequest filter topic as of 2019-01-23,
+	// when callback address, callback function, and expiration were added to the data payload.
+	RunLogTopic20190123 = utils.MustHash("RunRequest(bytes32,address,uint256,uint256,uint256,address,bytes4,uint256,bytes)")
 	// OracleLogTopic is the signature for the OracleRequest(...) event.
 	OracleLogTopic = utils.MustHash("OracleRequest(bytes32,address,uint256,uint256,uint256,bytes)")
```

Comment:
`OracleLogTopic` doesn't seem to be used anywhere, so let's remove it. I only added it as a placeholder for you to tweak, and you've since added `RunLogTopic20190123` which is better.